src/features/recitation/components/RecitationBar.tsx

```
@@ -82,19 +80,20 @@
-        </ErrorBoundary>
+    <div className="fixed bottom-0 left-0 right-0 z-50 bg-background rounded-t-lg border w-full mx-auto flex flex-col gap-4 pt-0">
+      <ErrorBoundary>
+        <SliderSection
+          audioUrl={audioUrl!}
+          timings={timings}
+        />
+      </ErrorBoundary>
+      <div className="flex flex-col gap-4 px-6 pb-6">
         <ErrorBoundary>
           <div className="flex flex-row items-center justify-between mr-5">
             <p>{isNaN(currentTime!) ? "00:00" : timeFormatter(currentTime!)}</p>
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)